test/gpu/native/GpuTestCommon.chpl

```diff
@@ -0,0 +1,38 @@
+module GpuTestCommon {
+  use GpuDiagnostics;
+
+  config const printGpuDiags = false;
+
+  proc verifyDiags(um, aod, h2d=-1, d2h=-1, d2d=-1) {
```

Comment:
cd22d47558 does a rename similar to what you suggest.

I think you said something about this already somewhere, but having to pass `-M` everytime you `chpl` a test (instead of `start_test`) could get annoying real fast. We can consider adding this function into `GpuDiagnostics` directly.